### PR TITLE
85-copy-link

### LINKED ISSUE #85: Copy Link
Let users copy a spinshare link for charts, playlists and user pages.

### PR DESCRIPTION
**Pull request by TaraSophieDev**
closes #85

### CODE REVIEW

File: SpinShareClient/UserInterface/src/views/Chart/Detail.vue [hunk 1]
```diff
@@ -123,6 +123,11 @@
                             v-tooltip="t('general.report')"
                             @click="handleReport"
                         />
+                        <SpinButton
```
Comment (LauraWebdev) [inline SpinShareClient/UserInterface/src/views/Chart/Detail.vue]: Please place this right below the "openOnSpinShare" button

File: SpinShareClient/UserInterface/src/views/User/Detail.vue [hunk 2]
```diff
@@ -47,6 +47,11 @@
                             v-tooltip="t('general.report')"
                             @click="handleReport"
                         />
+                        <SpinButton
```
Comment (LauraWebdev) [inline SpinShareClient/UserInterface/src/views/User/Detail.vue]: Please place this right below the "openOnSpinShare" button

**Review by LauraWebdev — CHANGES_REQUESTED** [review]
Thank you for your contribution, I've added two change requests, will merge once they are fixed.

**Comment by TaraSophieDev** [comment]
fixed it.

**Comment by LauraWebdev** [comment]
Perfect, thanks!

**Review by LauraWebdev — APPROVED** [review]

**Pull request merged**